region/crateria/east.json

```diff
@@ -362,7 +362,12 @@
           "locks": [
             {
               "lockType": "permanent",
-              "unlock": [ "never" ]
+              "unlock": [ "never" ],
+              "bypass": [
+                "canCrystalFlash",
```

Comment:
Using crystal flash as a glitch tech feels quite different from just using it for health